cuttings/router.js

```diff
@@ -0,0 +1,100 @@
+const { Router } = require('express');
+const cuttingRouter = new Router();
+const Cutting = require('./model');
```

Comment:
first imports, then the rest of the code

Reply:
Great tips